docs/pages/Theming/ColorTokens/UsageWithComponentTokens.md

```diff
@@ -34,7 +34,7 @@ export const Foo = compose<TabItemType>({
       ['color', ...fontStyles.keys],
     ),
   },
-  render: (userProps: TProps, useSlots: UseSlots<TType>) => {
+  useRender: (userProps: TProps, useSlots: UseSlots<TType>) => {
```

Comment:
For my own understanding, this line defines a hook named `useRender` that is called inside the resulting components render? I had always assumed that this line created the _actual_ render function, not something used inside render.

Reply:
We call this function inside the component's render function. You can see it being called here:

https://github.com/microsoft/fluentui-react-native/blob/ae8453d3911de46140bd727559c787f56de67b67/packages/framework/composition/src/composeFactory.ts#L67